.gitlab-ci.yml

```diff
@@ -74,7 +74,9 @@ OSTree Images:
           - aws/rhel-8.7-ga-x86_64
           - aws/rhel-9.1-ga-x86_64
           - aws/rhel-8.8-nightly-x86_64
+          - aws/rhel-8.9-nightly-x86_64
           - aws/rhel-9.2-nightly-x86_64
+          - aws/rhel-9.3-nightly-x86_64
```

Comment:
Any reason why you didn't remove 8.8 and 9.2 nightly runners (like you did in the SonarQube job)? :thinking: